check_import.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+import sys
+import datetime
+
+# Replace with correct path
+FILE_PATH = 'testdir/test.csv'
```

Comment:
How to change it? Does it have to be hardcoded in this script?
I would prefer reading some environment variable, or reading a special config file, consuming a request parameter or something similar.